diff -r fbbf5e9d1fd1 -r cd3d09f4a5dd src/Makefile
--- a/src/Makefile	Thu Jul 23 15:12:04 2020 +0100
+++ b/src/Makefile	Fri Jul 24 16:09:07 2020 -0700
@@ -364,8 +364,8 @@
 	$(CC) $(CFLAGS) -c $< -o $@
 
 install:build
-	$(BSDINSTALL) -d -g0 -m0755 -o0 $(BINDIR)
-	$(BSDINSTALL) -g0 -m4755 -o0 $(TARGET1) $(BINDIR)
+	$(BSDINSTALL) -d -m0755 $(BINDIR)
+	$(BSDINSTALL) -m4755 $(TARGET1) $(BINDIR)
 ifeq "$(P12)" "1"
 	ln -sf $(BINDIR)/$(TARGET1) $(BINDIR)/p12
 endif
@@ -388,16 +388,16 @@
 	ln -sf $(BINDIR)/$(TARGET1) $(BINDIR)/p32
 endif
 ifeq "$(PIO)" "1"
-	$(BSDINSTALL) -g0 -m4755 -o0 $(TARGET2) $(BINDIR)
+	$(BSDINSTALL) -m4755 $(TARGET2) $(BINDIR)
 endif
 ifeq "$(PTEST)" "1"
-	$(BSDINSTALL) -g0 -m4755 -o0 $(TARGET3) $(BINDIR)
+	$(BSDINSTALL) -m4755 $(TARGET3) $(BINDIR)
 endif
 ifeq "$(PCTRL)" "1"
-	$(BSDINSTALL) -g0 -m4755 -o0 $(TARGET4) $(BINDIR)
+	$(BSDINSTALL) -m4755 $(TARGET4) $(BINDIR)
 endif
 ifeq "$(PLOAD)" "1"
-	$(BSDINSTALL) -g0 -m0755 -o0 $(TARGET5) $(BINDIR)
+	$(BSDINSTALL) -m0755 $(TARGET5) $(BINDIR)
 endif
 
 uninstall:
